### DISCUSSION TITLE
大家喜歡用紙本還是平板讀書？

### DISCUSSION THREAD

**Discussion by sjh501** [1101-GS4719-C/Python-Community-GS4719-C#1453] [General]
自從上大學看到周遭很多人都開始用平板上課、作筆記。我自己目前還是在使用實體的課本和筆記本，除了購買平板是一筆開銷外，自己試用過別人的平板後發現施力的方式和觸感和紙筆差別很大，還是熟悉的紙筆最對味；還有一點是現在遇到的大考基本上都還是用紙筆測驗，練一下手感總是在考試時會順一些。不過隨著課本越來越厚、每學期都用三四包的紙，感覺好像終究是要回到平板。

**Comment by zephyrxin01** [comment]
平板在處理筆記上相對來說比較方便，尤其是錯題或是重點整理真的比起紙本有很大的優勢。雖然我也還是比較喜歡紙上書寫的感覺，就算是類紙膜也無法取代筆尖壓進去紙上的感覺。

**Comment by ziihui** [comment]
喜歡用平板讀書，自從用了平板之後就不用擔心找不到筆記，要翻回去查看之前學過的攻勢，也超級方便!直接在平板下載老師上課的ppt，現在上課大多只要帶平板出門，超級快樂!

**Comment by 1104030360** [comment]
我還是比較喜歡用紙本看書的感覺，因為摸到實體的紙張才會讓我有自己真的在看書的feel，也更容易讀得進腦袋裡。

**Comment by JessyCheng2003** [comment]
我喜歡用平板讀書，我覺得平板讀書很方便，尤其是在抄筆記的時候，如果要複製東西，直接按複製就好，但是如果是紙本就需要重寫一次，還有，如果今天要移動筆記或是圖案，在平板可以直接選取起來就移動，但是是紙本的話，就需要擦掉重新寫一次。但是如果是寫日記的話，我還是喜歡用紙本，因為紙和筆比較有溫度～

**Comment by Chiuhsianglin** [comment]
剛進大學時還沒買平板，上課的時候都需要帶著原文書還有自己的筆記本趴趴走，但自從有了平板後，書包輕了許多，而且我認為平板的操作非常方便，可以有系統地整理，也不需要額外印老師的ppt，直接在平板上面做筆記就可以了。

**Comment by ashleyy-huang** [comment]
對我來說使用平板最方便的是可以有系統地整理所有上課PPT，且不用常常需要跑影印店來列印上課教材，也不用需要背著厚厚的一本書去上課，但如果是論寫起來的手感，我還是喜歡寫在紙上遠大於電子書，不過為了方便犧牲一些手感對我來說不是什麼大問題

**Comment by alstinlin** [comment]
習慣之後覺得平板比書本方便好多，比如說用平板筆記會比用書本少了塗改的痕跡，所以變得更整齊，而且老師幾乎都會把教材傳到網路上給我們下載，就不需要再帶厚重的書本去上課了

**Comment by pudding0803** [comment]
雖然我沒有平板🤣，不過我覺得平板感覺會很方便。除了筆記可以變得非常彈性，需要突然加什麼東西或是移動之前的筆記，都可以輕鬆做到，對於我這種有強迫症的感覺非常適合🥹。不過紙本寫起來的感覺很真實(?)，而且筆記如果認真做，看起來會很有成就感🥳

**Comment by ALAN30359704** [comment]
我認為用平板寫字其實蠻花時間的，因為常常在畫力學的圖的時候常常需要花自由體圖，所以會導致摩擦力不足，直線也沒有辦法成一直線。而且使用平板的時候眼睛常常需要長時間的看著螢幕，所以眼睛會非常的酸痛。

**Comment by Tzuhsuannnnn** [comment]
我認為平板十分方便，雖然一開始的我也擔心會不會不習慣在平板上寫字，但其實很快就習慣了，不但少了多在文具上的開銷，也不用買教科書或將教授PPT印出來，可以直接在平板上做筆記或寫功課，我覺得十分方便，也可以減少紙的用量，愛護地球。

**Comment by Superlan8** [comment]
我還是紙本派，雖然身邊越來越多人開始用平板，也開始推坑我平板的好，但有兩個原因是我目前不會離開紙本的，第一，價錢因素，平板及觸控筆都是一筆耗費，相較於紙本來說，費用昂貴許多，第二，習慣，從小習慣在紙本上書寫，在平板寫字對我來說還不太適應，對我來說，紙本對於記憶也比較有效，因此目前我還是在用紙本讀書。

**Comment by Miinteaa** [comment]
我是平板派！因為原文書真的太太太重了，使用電子檔代替的話可以免去攜帶的困擾，且以檔案儲存的方式也不會放在書櫃上生灰塵，我覺得很棒。另一個使用平板上課的益處是可以直接擷取畫面上或者網路上的圖樣作為輔助學習，就不用另外影印或是手繪，更能達到環保的目的♻️

**Reply by Miinteaa** [reply]
不過若是要閱讀課外讀物的話，我還是喜歡紙本摸起來的手感，以及收藏自己喜愛的書籍的感覺！

**Comment by pigeon9975** [comment]
平板方便的點在於很輕，就算整天滿堂還是只要帶一台平板在身上就可以應付各種需求，而且電子書的檔案也可以免費下載，減少不少支出，但缺點就是平板用久了眼睛會很不舒服，而且就算有貼類紙膜，我覺得書寫感還是和真實的紙有蠻大的差異，所以，像通識或是系上那種比較不需要計算的課，我就會使用平板上課、複習，至於像微積分、向量分析等需要一直寫一直算的課，我還是寧願背很多東西也願意使用實體課本。

**Comment by Aaron0810** [comment]
平板+1，我是大三的，剛開始進大學的時候還沒有平板，那時候我們系上也不太有人在用平板，所以就跟著大家買原文書用紙本，但每次去上課都很麻煩，書包超重，後來上大二才買了平板，自從買了平板之後就完全不想帶書了，超級方便。我覺得用紙本沒有不好，因為那會讓你更珍惜自己曾經做過的筆記，但用平板之後真的就完全回不去了。

**Comment by Aaron25564448** [comment]
單論做筆記的話，我認為的確平板在修改、排版方面是方便許多，而且也比較輕便。但如果是閱讀書籍的話，我還是偏好實體書，剛買的新書通常都會有個神奇的味道，我也比較習慣翻書的手感。

**Comment by alicia0725** [comment]
只說筆記的話我個人比較喜歡用紙筆，感覺學起來記憶比較深刻，但我覺得如果是上課來說 ，用平板真的可以代替笨重的課本及紙質講義

**Comment by aa103967622** [comment]
我自從用了平板就回不去了，每天上課背一堆原文書覺得重得要命，直接在平板儲存每一科的電子書真香！而且也可以把教授的PPT載下來在上面做筆記📝，不得不說真的超方便！原本我比較習慣在紙本上書寫的感覺，但是在我買了類紙膜後，我在平板上也可以體驗到在紙上唰唰唰的觸感，另外，很多軟體有畫圖的輔助功能，直線、圓形和曲線都隨便自己畫，還有很多很多軟體，有著許多不同的功能呢！

**Comment by brian431** [comment]
個人認為若是需要經常在不同的書當中切換，又或是要帶來帶去的，還是用平板較為適合。還有若是需有做筆記的話用平板也是比較方便。

**Comment by lyien1021** [comment]
以前我也是覺得用紙筆就好，但自從買了平板，就回不去了~因為搭配筆記app，可以把所有科目筆記彙整在同個地方真的太方便，而且筆記也可以更多元化，像是插入圖片影片之類的。

**Comment by Brownnnn78** [comment]
我自己本來是沒有偏好哪一種，平板對於我來說就是放老師給的講義，還有可以帶出去讀書的時候不用帶著重重的課本，所以只能說是方便而不是偏好，但這學期開始想回味紙筆的感覺所以改用紙筆，最大的差異就是可以很一目了然的看到自己筆記的整體，不像在平板內一次只能看到一小區塊而已，平板真的變成只拿來放講義的螢幕了。

**Comment by juliaouo** [comment]
我會使用平板，首先它不會讓我的書包帶一大堆紙導致非常重，而且對於資料的整理非常方便，不會像紙本需要翻找半天，還有可能會不見。並且在做筆記時還能順便錄音或把照片拍下來，對於後續的複習非常有幫助！對於喜歡手寫質感的人也可以貼類紙膜，在不做筆記時還能作為娛樂工具，唯一的缺點大概就是沒電的時候會有點麻煩吧！

**Comment by 111607004** [comment]
平板派+1，實體書真的太重，而且很多老師上課都會使用簡報或電子檔講義，直接把簡報或講義下載下來在上面做筆記真的非常方便，有些重要的圖片也可以直接在課堂上拍照，並直接用平板註記，平板真的是天才發明！！！

**Comment by xiang305** [comment]
我也是比較喜歡用平板，有時候筆記內容很多，又或是老師補充了什麼重點，用紙本筆記寫的話就會空間不太夠。不過用平板的話就可以用app的工具去移動字的位置或放大和縮小。我個人是覺得方便多了

**Comment by Elaine2899** [comment]
我的話絕對是平板派，因為我常常寫錯或者不滿意自己寫的內容(包含大小、位置等等)，而平板能提供我更優良的改善空間，讓我可以在抄筆記時不會需要一直用橡皮擦擦拭或是用立可帶立掉不滿意的內容，一來減少許多時間，二來增加我的效率。另外，平板提供了多工的功能，讓我可以一邊在老師的投影片上做筆記，一邊在課本上做筆記，而書面無法做到這點的原因主要是學校桌子空間太小，放一本原文書都不夠，來要放鉛筆盒、水壺等等更是不可能。所以我更喜歡用平板讀書，也比較好少到自己抄錄的筆記。

**Comment by Evening218** [comment]
平板對於需要在老師的ppt上直接做筆記真的很方便,而且也比較適合喜歡排版自己的筆記的人,而且也能更好的保留筆記不會像紙本很難保存,但是對於需要大量計算的東西我認為紙本還是比較好

**Comment by MaxChi870** [comment]
平板的使用確實相較紙筆方便許多，也較一般的筆記紙易保存，但平板的購買是一大經濟問題負擔，且習慣紙筆的觸感，所以仍是使用傳統的紙筆作筆記。

**Comment by Alex11392** [comment]
我自己沒有平板，但上課和準備考試使用老師的ppt就足夠了，所以使用筆電就可以，不用平板也不用帶厚重的書

**Comment by ThanatosJun** [comment]
我現在是雙管齊下，一半用平板，一半用筆記課本。對於前者我認為有兩項優點，一是攜帶方便，二是彙整方便；但也有兩個缺點，一是電力問題，二是眼睛看太久很酸。後者同樣有兩項優點，一是手感好，二是眼睛不會比看平板時酸；壞處也有兩項，最嚴重的是重量與體積，超不方便，另外就是彙整筆記東一塊西一塊，難調整。

**Comment by 110302001** [comment]
我還是喜歡用紙本～尤其看到自己漂亮整齊的筆記就會很有成就感，但我自已用平板的字寫起來就沒那麼好看

**Comment by lin157644** [comment]
我個人筆記是使用平板，平板可以直接在老師的投影片上寫筆記，而且我覺得複習時比較好回想上課時的內容，但是教科書因為需要經常來回查閱或是前後對照，用平板就覺得不太方便，因此教科書我更偏好使用紙本的。
